npg-api/npg-notification-api.yaml

```diff
@@ -0,0 +1,114 @@
+
+openapi: 3.0.3
+info:
+  title: pagoPA Wallet Notification API
+  version: "0.1"
+  description: |-
+    Notification API to implement callback providing order status updates after wallet creation on Nexi's NPG
+externalDocs:
+  description: Nexi API to handle Hosted Payment Page (HPP) payments
+  url: https://developer.nexigroup.com/it/api/post-orders-hpp
+paths:
+  /notify:
+    post:
+      summary: Update Wallet on NPG payment instrument creation
+      description: Update Wallet on NPG payment instrument creation
+      operationId: notifyWallet
+      parameters:
+        - in: header
+          name: Correlation-Id
+          schema:
+            type: string
+            format: uuid
+          required: true
+          description: Unique identifier of the request
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/NotificationRequest'
+        required: true
+      responses:
+        '204':
+          description: Notification handled successfully
+        '400':
+          description: Invalid input
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProblemJson'
+        '401':
```

Comment:
This status code is to be returned in case of mismatch between received security token and saved one?

Reply:
Exactly :)